modelbaker/dbconnector/pg_connector.py

```diff
@@ -1020,11 +1021,28 @@ def create_basket(self, dataset_tid, topic, tilitid_value=None):
                     topic
                 )
             try:
+                print(tilitid_value)
```

Comment:
Here as well

Reply:
check if you are doing it right ;-) 

No, I forgot. Thanks.